pyartcd/pyartcd/pipelines/check_bugs.py

```diff
@@ -0,0 +1,230 @@
+import os
```

Comment:
some tests would be great

Reply:
done here: https://github.com/openshift/aos-cd-jobs/pull/3003/commits/7e42210bf38259cb2f3bc265ac07f4855700a287

Reply:
Done here: https://github.com/openshift/aos-cd-jobs/pull/3003/commits/4f6bd28a0cd4a104b3807e5cb38bad4a26f87492